src/battle-dex-data.ts

```diff
@@ -1177,7 +1177,7 @@ class Move implements Effect {
 				} else if (this.basePower >= 45) {
 					this.gmaxPower = 75;
 				} else  {
-					this.gmaxPower = 10;
+					this.gmaxPower = 70;
```

Comment:
Does this need to be fixed on server, too, or is it implemented differently there?

Reply:
Looks like SadisticMystic fixed it here, but also fixed some other things which should probably be propagated here.

https://github.com/smogon/pokemon-showdown/commit/b2e9336c327efbd22d36eb69f9b7905cd3955fb4